examples/mytask.rb

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
```

Comment:
I ended up adding a unit test for the example as a demonstration of testing your own task. I think that'd be useful to include.